apps/bundle_deploy/crt_config/crt_config.h

```diff
@@ -35,9 +35,9 @@
 /*! Maximum supported arguments in generated functions */
 #define TVM_CRT_MAX_ARGS 10
 /*! Maximum supported string length in dltype, e.g. "int8", "int16", "float32" */
-#define TVM_CRT_STRLEN_DLTYPE 10
+#define TVM_CRT_MAX_STRLEN_DLTYPE 10
```

Comment:
There is still a `crt/` in `template_project` besides `crt_config/`, I believe the former should be deleted?

Reply:
good catch, i agree.